.vscode/launch.json

```diff
@@ -26,11 +26,45 @@
       },
       "args": [
         "--mode=getEnviroData",
+        "--kind=vcast",
         "--clicast=C:/vcast/vc23sp2/clicast.exe",
         "--path=c:/RDS/SandBox/VectorCAST/examples/unitTests/FOO"
       ],
       "console": "integratedTerminal"
     },
+    {
+      "name": "getCoverageData",
+      "type": "python",
+      "request": "launch",
+      "program": "${workspaceFolder}/python/vTestInterface.py",
+      "cwd": "C:/RDS/SandBox/VectorCAST/examples",
+      "env": {
+        "PYTHONPATH": "C:/vcast/vc22sp7/python"
+      },
+      "args": [
+        "--mode=getCoverageData",
+        "--kind=vcast",
+        "--clicast=C:/vcast/vc23sp2/clicast.exe",
+        "--path=c:/RDS/SandBox/VectorCAST/examples/unitTests/FOO"
+      ],
+      "console": "integratedTerminal"
+    },
+    {
+      "name": "getCompletion",
+      "type": "python",
+      "request": "launch",
+      "program": "${workspaceFolder}/python/testEditorInterface.py",
+      "cwd": "C:/RDS/VectorCAST/testExplorer/unitTests",
+      "env": {
+        "PYTHONPATH": "C:/vcast/vc23/python"
```

Comment:
What's with these changes?

Reply:
These were removed in ea4948a8e6419375b1d8b874586b4a4f313eca86 . They ended up here when I was branching from a stable version and dealing with merge conflicts. As they seemed to be unused commands now, I can get rid of these

Reply:
Ah, yeah, would be good to remove these